agent/agent.go

```diff
@@ -208,6 +208,8 @@ func formatBytes(val uint64) string {
 
 func handle(conn io.ReadWriter, msg []byte) error {
 	switch msg[0] {
+	case signal.StackTraceLegacy:
+		return pprof.Lookup("goroutine").WriteTo(conn, 1)
```

Comment:
Instead of adding a new command, would there be a way to pass the `debug` argument to the agent as part of the existing `signal.StackTrace` command? If this is not possible in a backwards-compatible way, I'd suggest instead to add a new more generic command which allows to pass the `debug` value as part of the command invocation, so users could e.g. use `debug=0` to write in compressed format.